setup/helpers/perf_helper.rb

```diff
@@ -21,20 +25,26 @@ def install_epel_packages
     # package for the metrics stuff.
 
     step 'add epel' do
-      # Graphite / grafana needs a newer version of python which is only found in the epel repo
+      # Graphite / grafana needs a newer version of python which is only found
+      # in the epel repo.
       # Also needed for newer version of atop
       hosts.each do |host|
         platform_ver = host['platform'].version
-        epel_url = "https://dl.fedoraproject.org/pub/epel/epel-release-latest-#{platform_ver}.noarch.rpm"
-        host.install_package(epel_url, '', nil, :acceptable_exit_codes => [0,1])
+        fedora_ep = 'https://dl.fedoraproject.org/pub/epel'
+        epel_url = "#{fedora_ep}/epel-release-latest-#{platform_ver}.noarch.rpm"
+        host.install_package(epel_url, '', nil, acceptable_exit_codes: [0, 1])
       end
     end
 
     step 'disable selinux on metrics server' do
       # disable selinux immediately, but not persistent after a reboot
       on agents, 'setenforce 0 || true'
       # required to disable selinux between reboots
-      on agents, "sed -i 's/^SELINUX=.*/SELINUX=disabled/g' /etc/sysconfig/selinux && cat /etc/sysconfig/selinux" unless
+      sed_cmd = "sed -i 's/^SELINUX=.*/SELINUX=disabled/g' "\
+                '/etc/sysconfig/selinux '\
+                '&& cat /etc/sysconfig/selinux'
+      on agents, sed_cmd unless
+          # FIXME: What is `agents[0]` intended to be?
```

Comment:
I think this was just a catch for in case hosts were centos6...  It assume all hosts are the same os though... probably should loop through all agents and run the sed command on anything that isn't centos6 instead.

Reply:
of course the comment says only on the metrics box... so I guess it is assuming there is only one agent and it is the metrics box.  :(